### ISSUE TITLE
Migrate search input component to UI-kit

### ISSUE THREAD

**Issue by Rombelirk** [commercetools/ui-kit#1618]
**Is your feature request related to a problem? Please describe.**
The `Search input` component that is used in Products Filters is going to be also used in Orders Filters and Customers Filters and could be migrated to the ui-kit from the shared packages folder. (Similar to the `SearchSelectInput`). 
https://github.com/commercetools/merchant-center-frontend/tree/master/packages-shared/core/components/fields/search-input

**Comment by jonnybel** [comment]
Would the new component that @mohib0306 added serve the purpose of that component?
https://uikit.commercetools.com/?path=/story/components-inputs-selectinputs--searchselectinput

**Comment by Rombelirk** [comment]
Hmm, I guess not, because it's more about selecting one option out of a set of suggestions. And the search input is just about typing stuff and pressing the search button. But yeah, maybe that would make sense to just extend the existing component with that functionality instead of creating a bunch of slightly different ones. @tdeekens what do you think?

**Comment by tdeekens** [comment]
Yeah, we're more in need for a `SeachInput` than a `SearchSelectInput` as we'd like to use a UIKit component above any table which as a search input. This component is currently only internal to the MC but we felt eventually it would be something UIKit could house.

**Comment by jonnybel** [comment]
I see, thanks for the info!
This will be under discussion and will probably be added to our roadmap.

Meanwhile, perhaps it's best to move it to shared/experimental on the MC, so you can already use it across the different apps, and rework it (if needed) to be consistent with the API of similar ui-kit components, so that later it can then be moved seamlessly to the UI-kit with minimal adjustments.

**Comment by tdeekens** [comment]
Thanks, yes that was our idea too. We just wanted to get both things moving at the same time :)

**Comment by adnasa** [comment]
Setting this to `on hold` given that SHIELD are focusing on other topics
Will include it as part of `v13` nice to have

**Comment by mmaltsev-ct** [comment]
@FilPob todo: follow-up regarding existing guidelines for this component. Thanks!
